src/OBO.NET/DBXref.fs

```diff
@@ -1,16 +1,40 @@
 ﻿namespace OBO.NET
 
 
+open FSharpAux
+open ControlledVocabulary
+
 open System
 
 
-/// Representation of dbxrefs.
-type DBXref = {
+/// Representation of DBXrefs.
+type DBXref = 
+    {
         Name        : string
         Description : string
         Modifiers   : string
     }
 
+    /// Parses a given string to a DBXref
+    static member ofString (v : string) =
+        let xrefRegex = Text.RegularExpressions.Regex("""(?<xrefName>^([^"{])*)(\s?)(?<xrefDescription>\"(.*?)\")?(\s?)(?<xrefModifiers>\{(.*?)}$)?""")
+        let matches = xrefRegex.Match(v.Trim()).Groups
+        {
+            Name        = matches.Item("xrefName")          .Value |> String.trim
+            Description = matches.Item("xrefDescription")   .Value |> String.trim
+            Modifiers   = matches.Item("xrefModifiers")     .Value |> String.trim
+        }
+
+    /// Returns the corresponding CvTerm of the DBXref with empty name.
+    member this.ToCvTerm() = {
+        Name        = ""
+        Accession   = this.Name
+        RefUri      = 
+            String.split ':' this.Name

```

Comment:
Is this reliable? Will all TAN have this shape? Just curious, i don't know the answer myself

Reply:
[The OBO Flat File Format section](https://owlcollab.github.io/oboformat/doc/GO.format.obo-1_4.html#S.1.6) defines it this way, but with _SHOULD_.
I, myself, _should_ therefore add a `try with` there. 😉

Reply:
Nvm., `String.split` does not throw if there's not the given delimiter. It just returns an array with the input string as single item:

```fsharp
String.split ':' "klsdklsd"

val it: string[] = [|"klsdklsd"|]
```

Therefore, no need to do anything here, and I'm fine with this output for IDs that don't follow the format recommendations.